src/utils/updateNickname.ts

```diff
@@ -0,0 +1,28 @@
+import { INTERNAL_SERVER_ERROR } from "../constants/responses";
+import { DISCORD_BASE_URL } from "../constants/urls";
+import { env } from "../typeDefinitions/default.types";
+
+export async function updateNickName(
+  discordId: string,
+  nickname: string,
+  env: env
+) {
+  const changeNickNameURL = `${DISCORD_BASE_URL}/guilds/${env.DISCORD_GUILD_ID}/members/${discordId}`;
+  const data = { nick: nickname };
+  try {
+    const nameChangeResponse = await fetch(changeNickNameURL, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bot ${env.DISCORD_TOKEN}`,
+      },
+      body: JSON.stringify(data),
+    });
+    if (nameChangeResponse.ok) {
+      return await nameChangeResponse.json();
+    }
```

Comment:
What happens if the response is not ok? 
Are we not handling that case?

Reply:
it should throw an error in that case which would be caught in catch block, am I wrong here?

Reply:
Same question as @yesyash

Reply:
Hey @yesyash I have updated the PR.